sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.cpp

```diff
@@ -317,22 +411,20 @@ Error jit_compiler::linkDeviceLibraries(llvm::Module &Module,
   //       mandated by the extension.
```

Comment:
I'd argue we can drop the comment. With the new wrapper classes, this ties well into the overall RTC error reporting and I think getting the same diagnostic messages as the normal compilation flow makes it worth it.